src/prog_models/utils/calc_error.py

```diff
@@ -37,36 +34,24 @@ def MAX_E(m, times: List[float], inputs: List[dict], outputs: List[dict], **kwar
     Returns:
         float: Maximum error between model and data
     """
-    if isinstance(times[0], Iterable):
-        # Calculate error for each
-        error = [MAX_E(t, i, z, **kwargs) for (t, i, z) in zip(times, inputs, outputs)]
-        return max(error)
-
     x = kwargs.get('x0', m.initialize(inputs[0], outputs[0]))
-    dt = kwargs.get('dt', 1e99)
+    dt = kwargs.get('dt', 1e-99)
```

Comment:
I think this should still be 1e99. This change was propagated through all the error methods, and I think it's a mistake. But please confirm, I may be confused. 

Also, the tests currently aren't running to completion due to reaching the time limit. I can't run them on my local machine either. I think it's because the time step for many of the simulations is 1e-99